compiler/passes/convert-uast.cpp

```diff
@@ -1844,6 +1983,13 @@ struct Converter {
         const bool useLinkageName = false;
         conv = convertVariable(var, useLinkageName);
 
+        // Attach the doc comment if it exists.
+        DefExpr* defExpr = toDefExpr(conv);
+        INT_ASSERT(defExpr);
+        auto varSym = toVarSymbol(defExpr->sym);
+        INT_ASSERT(varSym);
+        varSym->doc = comment;
```

Comment:
Won't this attach the comment to all of the MultiDecl variables? Is that what is intended? I would think it would only go on the 1st of them.  You could do that by setting `comment = nullptr;` after using the comment here.

Reply:
I think in `buildVarDecls` we loop through and attach the doc comment to each `VarSymbol`, it would be nice to get a double check on that though. @lydia-duncan ?

Reply:
In practice, I suspect multi-variable declarations do not turn up much in documentation in practice - if the user wants them to have different documentation, they'll separate into single declarations; if the user wants the same documentation, they wouldn't have any recourse except copying unless we do as the code is currently written